### DISCUSSION TITLE
202212 | 保姆式安装DCE社区版 - DaoCloud Enterprise

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#871] [Announcements]
# 202212 | 保姆式安装DCE社区版 - DaoCloud Enterprise

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/blogs/dce5-install1209/

**Comment by jzhupup** [comment]
下载 dce5-installer 章节 建议连接到我们最新版本，目前0.3.28 存在很多已知问题

**Comment by windsonsea** [comment]
Well received. Added a note [at top](https://docs.daocloud.io/blogs/dce5-install1209/) to indicate only for reference.

**Comment by panpan0000** [comment]
这篇博客很多都是针对比较老的 dce5 版本，是否需要更新一版了？

**Reply by jzhupup** [reply]
赞同

**Reply by samzong** [reply]
已完成更新 https://docs.daocloud.io/blogs/new-dce5-free/
